environment/environment_blackbox_test.go

```diff
@@ -0,0 +1,71 @@
+package environment_test
+
+import (
+	"context"
+	"testing"
+
+	testsuite "github.com/fabric8-services/fabric8-common/test/suite"
+	"github.com/fabric8-services/fabric8-env/configuration"
+	"github.com/fabric8-services/fabric8-env/environment"
+	"github.com/fabric8-services/fabric8-env/resource"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type EnvironmentRepositorySuite struct {
+	testsuite.DBTestSuite
+}
+
+func TestEnvironmentRepository(t *testing.T) {
+	resource.Require(t, resource.Database)
+	config, err := configuration.New("")
+	require.NoError(t, err)
+	suite.Run(t, &EnvironmentRepositorySuite{DBTestSuite: testsuite.NewDBTestSuite(config)})
+}
+
+func (s *EnvironmentRepositorySuite) TestCreateEnvironment() {
+	envRepo := environment.NewRepository(s.DB)
+	newEnv := newEnvironment("osio-prod", "prod", "cluster1.com", uuid.NewV4())
+
+	env, err := envRepo.Create(context.Background(), newEnv)
+
+	require.NoError(s.T(), err)
+	assert.NotNil(s.T(), env)
+	assert.NotNil(s.T(), env.ID)
```

Comment:
Also add `Load(ctx, envID)` to the repository to load the env by its ID. Then in the test create an env. Load the env by its ID and check if they are equal.